CONTRIBUTING.md

```diff
@@ -89,3 +89,6 @@ Output format of any non-plumbing command can change in non-backward-compatible
 1. Thanks to the courtesy of [Ila&iuml; Deutel](https://github.com/ilai-deutel),
    a [git-machete package](https://aur.archlinux.org/packages/git-machete) is hosted in Arch User Repository (AUR).
    If the release introduces significant changes/critical bugfixes, please [flag the package as out of date](https://aur.archlinux.org/pkgbase/git-machete/flag).
+
+1. Verify if changes you made are reflected in corresponding medium articles: [Make your way through the git (rebase) jungle with Git Machete](https://medium.com/virtuslab/make-your-way-through-the-git-rebase-jungle-with-git-machete-e2ed4dbacd02) or [Git Machete Strikes again!](https://medium.com/virtuslab/git-machete-strikes-again-traverse-the-git-rebase-jungle-even-faster-with-v2-0-f43ebaf8abb0).
```

Comment:
Pls mention the paths to the blog Markdown files in this repository! Since it just mentions _the changes you made_, but without specifying the files